src/algebra/group/to_additive.lean

```diff
@@ -217,6 +217,7 @@ meta def tr : bool → list string → list string
 | is_comm ("pow" :: s)                := add_comm_prefix is_comm "nsmul"     :: tr ff s
 | is_comm ("npow" :: s)               := add_comm_prefix is_comm "nsmul"     :: tr ff s
 | is_comm ("zpow" :: s)               := add_comm_prefix is_comm "zsmul"     :: tr ff s
+| is_comm ("square" :: s)             := add_comm_prefix is_comm "even"      :: tr ff s
```

Comment:
```suggestion
| is_comm ("is" :: "square" :: s)             := add_comm_prefix is_comm "even"      :: tr ff s
```
funny that `even` is the additive version of `is_square`.
We don't have `square` yet, right? I would prefer the name `is_square`, since `square` looks like a noun.

Reply:
Yes, `is_square` vs `even` is a little funny.

I pushed your suggestion.  If it builds, it might also answer an issue that I have in a different PR, where I want `to_additive` to deal with `is_regular` but not touch `regular`!

Reply:
This PR building doesn't contain any occurrences of `is_square` yet, so it wouldn't be a good test. But yes, the same method should be used to translate `is_regular`.

Reply:
Yes, you are right.  I meant that it is a test for me to see that the syntax works out.  The other PR has usages of `is_regular` and of `regular`-withoout-`is`, so I am now waiting for CI on the other PR!